velox/exec/Driver.h

```diff
@@ -735,15 +735,27 @@ class SuspendedSection {
 
 /// Provides the execution context of a driver thread. This is set to a
 /// per-thread local variable if the running thread is a driver thread.
-struct DriverThreadContext {
-  const DriverCtx& driverCtx;
+class DriverThreadContext {
+ public:
+  explicit DriverThreadContext(const DriverCtx* driverCtx)
+      : driverCtx_(driverCtx) {}
+
+  const DriverCtx* driverCtx() const {
+    VELOX_CHECK_NOT_NULL(driverCtx_);
+    return driverCtx_;
+  }
+
+ private:
+  const DriverCtx* driverCtx_;
 };
 
 /// Object used to set/restore the driver thread context when driver execution
 /// starts/leaves the driver thread.
 class ScopedDriverThreadContext {
  public:
   explicit ScopedDriverThreadContext(const DriverCtx& driverCtx);
+  explicit ScopedDriverThreadContext(
+      const DriverThreadContext* _driverThreadCtx);
```

Comment:
```
s/_driverThreadCtx/driverThreadCtx/
```

Reply:
I think it will cover the thread_local driverThreadCtx variable which is going to be used in the method.